.github/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 13. AIRAC (1610) - Added new Strumble CTA to ARTCC High - thanks to @hsugden (Harry Sugden)
 14. AIRAC (1612) - Runway track updates at Cardiff (EGFF), Edinburgh (EGPH) and Aberdeen (EGPD) - thanks to @hsugden (Harry Sugden)
 15. AIRAC (1611) - Cardiff (EGFF) RNAV approaches and associated airfield fixes - thanks to @hsugden (Harry Sugden)
+16. AIRAC (1610) - Doncaster (EGCN) Hold G1 Added, Removed stands 11 and 15
```

Comment:
Feel free to add your details at the end too! I can't actually work out your name from your username! 😄